modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -21,10 +21,16 @@ cp -rv /var/www/owncloud /opt/backup/owncloud && mysqldump <db_name> > /opt/back
 [[review-third-party-apps]]
 == Review Third-Party Apps
 
-Review any installed third-party apps for compatibility with the new
-ownCloud release. Ensure that they are all disabled before beginning the
-upgrade. After the upgrade is complete re-enable any which are
-compatible with the new release.
+Review any installed third-party apps for compatibility with the new ownCloud release. 
+You can find a list of all third-party apps by running the following command, from the root directory of your installation.
+
+[source,console]
+----
+sudo -u www-data ./occ app:list --shipped=false
```

Comment:
This is just a info:
`--shipped` does not do what we think is does. Shipped means that apps can/cant be disabled. It does not mean that apps are shipped by the tar file. There is a core pr (https://github.com/owncloud/core/pull/34563) in progress that addresses this issue. When merged, a new PR identifying tared apps needs to be added to `app:list`. Then we have to adopt that command here. For the time being, I would let this as written, better than nothing...

Reply:
Thanks for the clarification, @mmattel.